depends/packages/zeromq.mk

```diff
@@ -13,7 +13,9 @@ endef
 
 define $(package)_preprocess_cmds
    patch -p1 < $($(package)_patch_dir)/0001-fix-build-with-older-mingw64.patch && \
-   patch -p1 < $($(package)_patch_dir)/0002-disable-pthread_set_name_np.patch
+   patch -p1 < $($(package)_patch_dir)/0002-disable-pthread_set_name_np.patch && \
+   patch -p1 < $($(package)_patch_dir)/0003-new_configure.patch && \
+   ./autogen.sh
```

Comment:
Is it definitely necessary to re-run autogen.sh? Afaik configure just calls out to config.guess/config.sub. Unless there's some weird globbing involved, I should think that replacing those files would be enough.